style.css

```diff
@@ -173,6 +174,21 @@ recent work section style
   border: 1px solid #8993a4;
 }
 
+form button {
+  padding: 12px;
+  margin: 0 98px;
+  width: 131px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: var(--color-Safety-Orange);
+  border: 1px solid var(--color-Safety-Orange);
+  color: var(--color-white);
+}
+
 .Recent-work button {
   width: 124px;
 }
```

Comment:
Kindly ensure that you implement button interactions such as hover, active (when it's pressed), disabled and enable (default state) t match the Figma template.